heron/common/src/python/argparser.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2016 Twitter. All rights reserved.
```

Comment:
done, changed argparser.py to heronparser.py , 
changed HeronRCArgumentParser to HeronArgumentParser